getting-started/display.rst

```diff
@@ -49,6 +49,12 @@ Next, start up a Python interpreter.
 You can use the default Python shell (:command:`python`), the `IPython shell`_, or even run from a `Jupyter Notebook`_.
 Ensure that this Python session is running from the shell where you ran :command:`setup lsst_distrib`.
 
+Some user-reported solutions to ds9-related issues include:
```

Comment:
Please don't use the phrase "user-reported solutions" in our official documentation. It implies that we haven't tested it and validated it and doesn't provide any useful information.

Instead use the phrase, "If the image does not display with DS9, try these troubleshooting steps:"

Reply:
Also choose whether DS9 is all lower case or all uppercase. There should be consistency in the documentation.

Reply:
Updated wording to match yours